Rename Custom Reporting (#7151)
DataDog/documentation: content/en/account_management/billing/usage_attribution.md

---
title: Usage attribution
kind: documentation
aliases:
  - /account_management/billing/advanced_usage_reporting/
  - /account_management/billing/custom_usage_reporitng/ 
---

## Overview

Administrators can access the Usage Attribution tab from the Plan & Usage section in Datadog. The Usage Attribution page provides the following information and functionality:

* Lists the existing tag keys that usage is being broken down by and provides the ability to change and add new ones (up to three tag keys).
* Generates daily .tsv (tab separated values) files for most usage types.
* Summarizes usage at the end of each month. 
* Surfaces the data both in the UI and as a .tsv download.

> **Note**: The following usage types are not supported in this tool yet. They will be added subsequently:

Reviewer: ```suggestion
**Note**: The following usage types are not supported in this tool:
```
Style Guide - Docs don't mention things that could happen in the future.

* Indexed Log Events
* Ingested Logs
* Analyzed Spans

### Getting started

> In order to start receiving daily data, an administrator will need to create a new report via the user interface. 

Reviewer: ```suggestion
To start receiving daily data, an administrator needs to create a new report with the user interface. 
```
Style guide - no future tense, no use of the word `via` for translation purpose, `in order` is unnecessary

{{< img src="account_management/billing/advanced-usage-reporting-02.png" alt="Applied Tags" >}}

> * Once the tags are configured, it will take a full 24-hour period for the first report to be generated.

Reviewer: ```suggestion
* Once the tags are configured, it takes a full 24-hour period for the first report to be generated.
```

> * The reports will be generated on an ongoing basis.

Reviewer: ```suggestion
* The reports are generated on an ongoing basis.
```

* If tags are changed, the new reports will reflect the new tags. However, the previous reports will keep the old ones.
* Monthly reports reflect the latest set of tags. If you change tags mid-month, the usage percentages might not match up. 

### Monthly Usage Attribution

> Once the reports start being generated, they will be updated daily and aggregated monthly in this table.

Reviewer: ```suggestion
Once the reports start being generated, they are updated daily and aggregated monthly in this table.
```

{{< img src="account_management/billing/advanced-usage-reporting-03.png" alt="Usage Summary Table" >}}

* Clicking on a specific tag key updates the table view accordingly.
* If multi-org is enabled, usage is summarized across all Datadog organizations at the parent account.
* Previous months' reports are accessible through the time selector.
* Monthly reports are not generated until the month is over. Each monthly report should appear by the second day of the following month. 
* Reports are downloadable using the 'Export Current View' reports. These .tsv reports include both usage numbers and percentages, allowing for easy allocations and chargebacks.

### Daily Usage Attribution

This section provides daily reports at an hourly granularity to dig into timeframes. It also provides a concatenation of all reports during a given month.

> * Clicking on a specific time period will expand a view on the right where reports can be downloaded as a .tsv.

Reviewer: ```suggestion
* Clicking on a specific time period expands a view on the right where reports can be downloaded as a `.csv`.
```
